### PR TITLE
[UI] Fixed bug of settings page

### LINKED ISSUE #6956: [UI] Settings inconsistent with the UI,dashboard icon disappearing
#### Current Behavior
The UI of the settings page is not consistent with the UI also the dashboard icon was disappearing with the click.

#### Desired Behavior
The UI must be consistent for all pages.

#### Screenshots/Logs
Screenshot from 2023-01-31 01-12-24
Screenshot from 2023-01-31 01-12-31
a7d.png)

#### Environment

- Browser: Chrome Safari Firefox
- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by partik03**
**Notes for Reviewers**

This PR fixes #6956 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@partik03, it'd be helpful if you can remove any non-material changes from this PR (the whitespacing changes are a distraction and unnecessary).

**Comment by leecalcote** [comment]
@partik03 some great feedback awaits you in Slack.

File: Makefile [hunk 1]
```diff
@@ -28,7 +28,7 @@ docker-build:
 docker-playground-build:
 	# `make docker-playground-build` builds Meshery inside of a multi-stage Docker container.
 	# This method does NOT require that you have Go, NPM, etc. installed locally.
-	DOCKER_BUILDKIT=1 docker build -f install/docker/Dockerfile -t layer5/meshery --build-arg TOKEN=$(GLOBAL_TOKEN) --build-arg GIT_COMMITSHA=$(GIT_COMMITSHA) --build-arg GIT_VERSION=$(GIT_VERSION) --build-arg RELEASE_CHANNEL=${RELEASE_CHANNEL} --build-arg PROVIDER=$(LOCAL_PROVIDER) --build-arg PROVIDER_BASE_URLS=$(MESHERY_CLOUD_PROD) .	
+	DOCKER_BUILDKIT=1 docker build -f install/docker/Dockerfile -t layer5/meshery --build-arg TOKEN=$(GLOBAL_TOKEN) --build-arg GIT_COMMITSHA=$(GIT_COMMITSHA) --build-arg GIT_VERSION=$(GIT_VERSION) --build-arg RELEASE_CHANNEL=${RELEASE_CHANNEL} --build-arg PROVIDER=$(LOCAL_PROVIDER) --build-arg PROVIDER_BASE_URLS=$(MESHERY_CLOUD_PROD) .
```
Comment (leecalcote) [inline Makefile:31]: Please revert all changes to this file

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by Chadha93** [comment]
@partik03  Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by Chadha93** [comment]
@partik03 Could you please add a video/ss of the fix?

**Comment by partik03** [comment]
https://user-images.githubusercontent.com/99184676/216104667-503b956a-ccd1-4bad-a671-01dc3e6d1757.mp4

**Comment by acald-creator** [comment]
// @leecalcote @Abhishek-kumar09 

Looks great!

I would like others to chime in if they want to provide feedback.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks, @partik03! The icons used for the two menus need to be distinct. Will you please use this icon?
capabilities-registry


Be sure to include UTF-8 encoding in the svg, if it isn't there already.

**Comment by leecalcote** [comment]
merge conflicts

**Comment by partik03** [comment]
> Thanks, @partik03! The icons used for the two menus need to be distinct. Will you please use this icon? capabilities-registry
> 
> Be sure to include UTF-8 encoding in the svg, if it isn't there already.

I will fix this asap

**Comment by Chadha93** [comment]
@partik03 heads up.

**Comment by Chadha93** [comment]
@partik03  Let's discuss the update of this PR on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by partik03** [comment]
> Thanks, @partik03! The icons used for the two menus need to be distinct. Will you please use this icon? capabilities-registry
> 
> Be sure to include UTF-8 encoding in the svg, if it isn't there already.

Sorry but which two menus are you referring to??

**Comment by leecalcote** [comment]
@partik03 the only two that have the same icon.

File: ui/components/MeshModelComponent.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import {  withStyles } from '@material-ui/core'
+import { withSnackbar } from 'notistack';
+import React from 'react'
+const meshmodelStyles = (theme) => ({
+  wrapperClass : {
+    padding : theme.spacing(2),
+    backgroundColor : "white",
+    height : "60vh",
```
Comment (leecalcote) [inline ui/components/MeshModelComponent.js:8]: Why so huge?

File: ui/utils/utils.js [hunk 3]
```diff
@@ -99,18 +99,24 @@ export function getComponentsinFile(file) {
...
+  console.log("Pattern",pattern);
   const services = pattern[0]?.services;
   if (!services) {
     return { err : "Services not found in the design" };
   }
 
   const validationPayloads = {};
-
-  for (const serviceId in services) {
+  console.log("Services",services);
```
Comment (leecalcote) [inline ui/utils/utils.js:109]: console.debug here would be good.

Comment (partik03) [reply ui/utils/utils.js:109]: I would remove this code and might file another pr as I tried to address another issue [6715](https://github.com/meshery/meshery/issues/6715)

File: ui/components/MeshModelComponent.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  dashboardSection : {
+    backgroundColor : "#fff",padding : theme.spacing(2),borderRadius : 4,height : "100%",overflowY : "scroll"
+  },
+
+})
+const MeshModelComponent = ({ showMeshModelSummary,classes }) => {
+  return (
+    <div className={classes.wrapperClass}>
+      <div className={classes.dashboardSection} data-test="workloads">
+        {/* <Typography variant="h6" gutterBottom className={classes.chartTitle}>
```
Comment (MUzairS15) [inline ui/components/MeshModelComponent.js:17]: We can remove this part if no longer needed.

File: ui/components/MesheryPatterns.js [hunk 5]
```diff
@@ -1197,8 +1198,13 @@ function MesheryPatterns({
           open={modalOpen.open}
           handleClose={handleModalClose}
           submit={
-            { deploy : () => handleDeploy(modalOpen.pattern_file, modalOpen.name), unDeploy : () => handleUnDeploy(modalOpen.pattern_file, modalOpen.name) }
+            {
+              deploy : () => handleDeploy(modalOpen.pattern_file, modalOpen.name),  unDeploy : () => handleUnDeploy(modalOpen.pattern_file, modalOpen.name), verify : () => handleVerify(modalOpen.pattern_file, modalOpen.name)
+            }
           }
+          verify={() => {
```
Comment (MUzairS15) [inline ui/components/MesheryPatterns.js:1205]: Why is this needed??
You have provided `verify` callback here https://github.com/meshery/meshery/pull/6957/files#diff-e1d76eb522ad8ca1078d26e59d9c2635f97eeeaefdd95f7c693683fe3e151bd5R1202.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by acald-creator** [comment]
@partik03 Can you provide an update on this PR? Please take note of the other PR that is going to fix the Dark Mode option in this particular area of the MeshModel. https://github.com/meshery/meshery/pull/7073****

**Comment by partik03** [comment]
> @partik03 Can you provide an update on this PR? Please take note of the other PR that is going to fix the Dark Mode option in this particular area of the MeshModel. #7073****

@acald-creator I think I have made all the necessary changes and I think it can now be merged.But it will be great if you offer your review

**Comment by acald-creator** [comment]
// @partik03 Can you update your PR with the latest changes from master?

**Review by MUzairS15 — APPROVED** [review]

**Comment by partik03** [comment]
@acald-creator @leecalcote can you merge this one now??

**Comment by leecalcote** [comment]
@partik03 what new icon is being used?

**Comment by leecalcote** [comment]
@partik03 please execute these instructions - https://github.com/meshery/meshery/pull/6957/checks?check_run_id=11435431238

**Comment by partik03** [comment]
> @partik03 please execute these instructions - https://github.com/meshery/meshery/pull/6957/checks?check_run_id=11435431238

@leecalcote I am not able to signoff my commits now as this branch has got a lot of commits and rebasing it can cause serious collaboration problems,so can you please let the DCO thing go this time

**Comment by partik03** [comment]
> @partik03 what new icon is being used?

Its another icon from the fontawesome icons library

**Comment by leecalcote** [comment]
> > @partik03 please execute these instructions - https://github.com/meshery/meshery/pull/6957/checks?check_run_id=11435431238
> 
> @leecalcote I am not able to signoff my commits now as this branch has got a lot of commits and rebasing it can cause serious collaboration problems,so can you please let the DCO thing go this time

Ok, thanks for confirming, @partik03

**Pull request merged**